src/arch/cuda/hybridAPI/hapi.h

```diff
@@ -5,22 +5,6 @@
 /* See hapi_functions.h for the majority of function declarations provided
  * by the Hybrid API. */
 
-/******************** DEPRECATED ********************/
-// HAPI wrappers whose behavior is controlled by user defined variables,
-// which are HAPI_USE_CUDAMALLOCHOST and HAPI_MEMPOOL.
-#ifdef HAPI_USE_CUDAMALLOCHOST
-#  ifdef HAPI_MEMPOOL
-#    define hapiHostMalloc hapiPoolMalloc
-#    define hapiHostFree   hapiPoolFree
-#  else
-#    define hapiHostMalloc cudaMallocHost
-#    define hapiHostFree   cudaFreeHost
-#  endif // HAPI_MEMPOOL
-#else
-#  define hapiHostMalloc malloc
-#  define hapiHostFree   free
-#endif // HAPI_USE_CUDAMALLOCHOST
-
```

Comment:
This was retained because of ChaNGa, but ChaNGa will be updated such that this is not required (https://github.com/N-BodyShop/changa/pull/79).